scipy/stats/_stats_py.py

```diff
@@ -3748,6 +3748,14 @@ def f_oneway(*samples, axis=0):
     axis : int, optional
         Axis of the input arrays along which the test is applied.
         Default is 0.
+    equal_var: bool, optional
+        If True (default), perform a standard one-way ANOVA test that
+        assumes equal population variances [2]_.
+        If False, perform Welch's ANOVA test, which does not assume
+        equal population variances. Modified F-statistics and
+        dof will be calculated by each samples [4]_.
```

Comment:
```suggestion
        equal population variances [4]_. 
```
I think this is clear enough, right? The statistic and DOF are just part of performing Welch's test, right?